scss-lint.yml

```diff
@@ -160,8 +158,7 @@ linters:
     allow_element_with_id: false
 
   SelectorDepth:
-    enabled: true
-    max_depth: 3
+    enabled: false
```

Comment:
I'm not familiar with this rule so I'll defer, but I'd ask if increasing it is better than removing it?

Reply:
This rule is similar to `NestingDepth`, except this is simply for a single selector (so `.one .two .three` is 3 depth, and `.one .two .three .four` is 4).

Now that I'm thinking about it more, we may not necessarily want to disable this.  If we're trying to do some more complex styling where depth of applicability will matter, then this rule gets in the way, but that's likely more the exception than than norm.  And from my point above, frontend software is not the primary focus of our company, so we should pick rules that help suggest better practices for those less familiar, and those more familiar or writing styling intentionally in some way can just mark things as wontfix.

Thanks for asking!